src/pages/ExamsPage/index.test.jsx

```diff
@@ -1,31 +1,50 @@
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 
 import ExamsPage from '.';
 import * as hooks from './hooks';
 
+// nomally mocked for unit tests but required for rendering/snapshots
+jest.unmock('react');
+
 jest.mock('./hooks', () => ({
   useInitializeExamsPage: jest.fn(),
   useExamsData: jest.fn(),
 }));
 
-jest.mock('./components/ExamList', () => 'ExamList');
```

Comment:
we can't really mock out entire components so we're only rendering this alone like I was hoping to do with enzyme. Instead, I've opted to use data-testid in the child components this renders to avoid testing the implemention details of another component as much as possible. Recommending that as a pattern so we don't need to rework tests if one of those components changes.